### PR TITLE
[receiver/huaweicloudces]: Add huaweicloudces receiver

### PR DESCRIPTION
**Pull request by narcis96**
#### Description
Introduce a new receiver, huaweicloudces receiver. This is the first PR as skeleton for the receiver.

#### Link to tracking issue
Related to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/34953

#### Testing
N/A

#### Documentation
metadata.yaml, README.

### CODE REVIEW

**Comment by narcis96** [comment]
ping @ChrsMark
 @mwear

**Review by mwear — APPROVED** [review]

File: receiver/huaweicloudcesreceiver/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+type: huaweicloudcesreceiver
+
+status:
+  class: receiver
+  stability:
+    development: [metrics]
+  distributions: []
+  codeowners:
+    active: [heitorganzeli, narcis96]
```
Comment (ChrsMark) [inline receiver/huaweicloudcesreceiver/metadata.yaml:9]: Should this list include the sponsor as well?

Comment (narcis96) [reply receiver/huaweicloudcesreceiver/metadata.yaml]: done.

**Review by ChrsMark — APPROVED** [review]

**Comment by bogdandrutu** [comment]
https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/11914213987/job/33201687156?pr=36258

**Comment by narcis96** [comment]
@bogdandrutu all fixed

**Pull request merged**